src/components/AttachmentCarousel/CarouselActions/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Pressable} from 'react-native';
+
+const propTypes = {
+    /** handles onPress events with a callback  */
+    onPress: PropTypes.func,
```

Comment:
This also looks like it should not be optional since the prop is always provided.